libs/application/templates/new-primary-school/src/forms/Prerequisites.ts

```diff
@@ -0,0 +1,141 @@
+          title: newPrimarySchoolMessages.pre.externalDataSection,
+          subTitle: newPrimarySchoolMessages.pre.externalDataDescription,
+          checkboxLabel: newPrimarySchoolMessages.pre.checkboxProvider,
+          submitField: buildSubmitField({
+            id: 'submit',
+            placement: 'footer',
+            title: '',
+            refetchApplicationAfterSubmit: true,
+            actions: [
+              {
+                event: DefaultEvents.SUBMIT,
+                name: newPrimarySchoolMessages.pre.startApplication,
+                type: 'primary',
+              },
+            ],
+          }),
+          dataProviders: [
+            buildDataProviderItem({
+              provider: NationalRegistryUserApi,
+              title:
+                newPrimarySchoolMessages.pre.nationalRegistryInformationTitle,
+              subTitle:
+                newPrimarySchoolMessages.pre
+                  .nationalRegistryInformationSubTitle,
+            }),
+            buildDataProviderItem({
+              provider: ChildrenCustodyInformationApi,
+              title: '',
+              subTitle: '',
+            }),
+            buildDataProviderItem({
+              provider: UserProfileApi,
+              title: newPrimarySchoolMessages.pre.userProfileInformationTitle,
+              subTitle:
+                newPrimarySchoolMessages.pre.userProfileInformationSubTitle,
+            }),
+          ],
+        }),
+        buildMultiField({
+          id: 'hasNoChildren',
+          title: newPrimarySchoolMessages.pre.noChildrenFoundTitle,
+
+          children: [
+            buildDescriptionField({
+              id: 'noChildrenDecsription',
+              title: '',
+              description: newPrimarySchoolMessages.pre.noChildrenFoundReasons,
+            }),
+            buildDescriptionField({
+              id: 'noChildrenNationalRegistryDescription',
+              title: '',
+              marginTop: 3,
+              description:
+                newPrimarySchoolMessages.pre
+                  .noChildrenFoundNationalRegistryDescription,
+            }),
+            buildDescriptionField({
+              id: 'noChildrenIslandisInfo',
+              title: '',
+              marginTop: 2,
+              description:
+                newPrimarySchoolMessages.pre.noChildrenFoundIslandisDescription,
+            }),
+            buildDescriptionField({
+              id: 'noChildrenNationalRegistryInfo',
+              title: '',
+              marginTop: 3,
+              description:
+                newPrimarySchoolMessages.pre
+                  .noChildrenFoundNationalRegistryInfo,
+            }),
+            buildDescriptionField({
+              id: 'noChildrenMMSInfo',
+              title: '',
+              marginTop: 3,
+              marginBottom: 6,
+              description: newPrimarySchoolMessages.pre.noChildrenFoundMMSInfo,
+            }),
+          ],
+        }),
+      ],
+    }),
```

Comment:
This should rather be a warning/error in the dataCollection screen. 2-3 lines of info and then a link to more info if all of this needs to be accessible.

Reply:
This has been removed